storage/yolo-v4-tf_INT8/LICENSE

```diff
@@ -0,0 +1,19 @@
+MIT License
+
```

Comment:
```suggestion

Copyright (c) 2019 david8862


```

Reply:
License needs to match their posted license with copyright: https://raw.githubusercontent.com/david8862/keras-YOLOv3-model-set/master/LICENSE

Reply:
our own derived work. Followed pattern in other TPP / licenses.